bin/node/runtime/src/lib.rs

```diff
@@ -1907,7 +1913,7 @@ mod tests {
 
 	#[test]
 	fn perbill_as_onchain_accuracy() {
-		type OnChainAccuracy = <Runtime as onchain::Config>::Accuracy;
+		type OnChainAccuracy = pallet_election_provider_multi_phase::SolutionAccuracyOf<Runtime>;
```

Comment:
```suggestion
		type OnChainAccuracy = <<Runtime as pallet_election_provider_multi_phase::Config>::Fallback as NposSolver>::Accuracy
```

I know they are the same thing, but this is more correct. The point of this test it to test the `type Accuracy` that is used on `onchain` executions. `SolutionAccuracyOf<_>` might be something else in other runtimes.